tests/Unit/TestHelpers.hpp

```diff
@@ -283,3 +286,51 @@ std::string get_output(const Container& c) {
   os << c;
   return os.str();
 }
+
+/*!
+ * \ingroup TestingFramework
+ * \brief Alternative to CAPTURE that prints more digits.
```

Comment:
Perhaps "Catch's CAPTURE" so that people have some clue as to where to look for what CAPTURE is.